clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -874,9 +874,13 @@ public void subscribe(Pattern pattern, ConsumerRebalanceListener listener) {
     public void unsubscribe() {
         acquire();
         try {
+            //TODO: Reactivate the commented code once KAFKA-3491 is resolved
```

Comment:
I'd rather not keep the TODOs and the commented-out section of codes. As long as we have a JIRA to make sure we don't lose track of the issue, we can always look back at git history to see the changes.